src/OrchardCore/OrchardCore.Data/Documents/FileDocumentStore.cs

```diff
@@ -1,34 +1,30 @@
 using System;
-using System.IO;
-using System.Reflection;
-using System.Text.Json;
-using System.Threading;
+using System.Collections.Generic;
 using System.Threading.Tasks;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Options;
-using OrchardCore.Environment.Shell;
 using OrchardCore.Environment.Shell.Scope;
 
 namespace OrchardCore.Data.Documents
 {
     /// <summary>
-    /// A singleton service using the file system to store document files under the tenant folder, and that is in sync
-    /// with the ambient transaction, any file is updated after a successful <see cref="IDocumentStore.CommitAsync"/>.
+    /// An <see cref="IDocumentFileStore"/> using the <see cref="IFileDocumentStore"/>.
     /// </summary>
     public class FileDocumentStore : IFileDocumentStore
```

Comment:
Would you be able to remove `IFileDocumentStore`, I think we should just use `IDocumentStore` here, `FileDocumentStore` should just be a different implementation, like you already started to do by copying some logic from DocumentStore.

Maybe at the same time rename the `DocumentStore` to `DatabaseDocumentStore` to make it obvious.

Reply:
> Would you be able to remove `IFileDocumentStore`, I think we should just use `IDocumentStore` here, `FileDocumentStore` should just be a different implementation, like you already started to do by copying some logic from DocumentStore.

`IFileDocumentStore` is required by `FilePlacementsStore` (OrchardCore.Placements.FileStorage feature) and `FileContentDefinitionStore` (OrchardCore.Contents.FileContentDefinition feature), so I don't think we can get rid of it.